shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/communication/jdbc/connection/BackendConnection.java

```diff
@@ -93,6 +93,8 @@
     private final Map<String, StatementMemoryStrictlyFetchSizeSetter> fetchSizeSetters;
     
     private final AttributeMap attributeMap;
+
+    private boolean autoCommit = true;
```

Comment:
Do we need `volatile` here?

Reply:
A `BackendConnection` only be used by one session, Do not use `volatile`, performance should be better.